### PR TITLE
Docs: Summit Ascent Update

### PR DESCRIPTION
**Pull request by ax3l**
Update the Summit Ascent install location.

- [x] permission issue for directory to be fixed
- [x] rebuild with same GCC version (9.3.0) that we are using
- [x] rebuild with same HDF5 that we are using
- [x] runtime testing:
  - [x] iso-contours + particles
  - [ ] iso-contours + particles + volume rendering

### CODE REVIEW

File: Docs/source/install/hpc/summit.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ We use the following modules and environments on the system (``$HOME/warpx.profi
 
    # optional: Ascent in situ support
    #   note: build WarpX with CMake
-   export Ascent_DIR=/gpfs/alpine/world-shared/csc340/software/ascent/current/summit/cuda/gnu/ascent-install
+   export Ascent_DIR=/gpfs/alpine/csc340/world-shared/software/ascent/2021_08_26/summit/cuda/gnu/ascent-install/
```
Comment (ax3l) [inline Docs/source/install/hpc/summit.rst]: GCC 9.3.0 build:
```suggestion
   export Ascent_DIR=/gpfs/alpine/csc340/world-shared/software/ascent/2021_08_31/summit/cuda/gnu/ascent-install/
```

File: Docs/source/install/hpc/summit.rst [hunk 2]
```diff
@@ -59,7 +59,7 @@ We use the following modules and environments on the system (``$HOME/warpx.profi
 
    # optional: Ascent in situ support
    #   note: build WarpX with CMake
-   export Ascent_DIR=/gpfs/alpine/world-shared/csc340/software/ascent/current/summit/cuda/gnu/ascent-install
+   export Ascent_DIR=/gpfs/alpine/csc340/world-shared/software/ascent/2021_08_31/summit/cuda/gnu/ascent-install/
```
Comment (ax3l) [inline Docs/source/install/hpc/summit.rst]: With same HDF5 as ours:
```suggestion
   export Ascent_DIR=/gpfs/alpine/csc340/world-shared/software/ascent/2021_09_01_gcc_9_3_0_warpx/summit/cuda/gnu/ascent-install
```

**Review by lucafedeli88 — APPROVED** [review]
Thanks for this PR!

**Comment by ax3l** [comment]
`iso-contours + particles + volume rendering`:

Segfault in VTK-m:
```
11: /sw/summit/gcc/9.3.0-2/lib64/libstdc++.so.6(__cxa_throw+0x64) [0x20000ab57594]
    ?? ??:0

12: ./warpx() [0x12694ad0]
    vtkm::cont::cuda::internal::CudaAllocator::Free(void*) at ??:?

13: ./warpx() [0x12698c30]
    (anonymous namespace)::CudaDelete(void*) at ??:?

14: ./warpx() [0x12023ec0]
    vtkm::cont::internal::BufferInfo::~BufferInfo() at ??:?

15: ./warpx() [0x12018708]
    std::_Rb_tree<vtkm::cont::DeviceAdapterId, std::pair<vtkm::cont::DeviceAdapterId const, (anonymous na
mespace)::BufferState>, std::_Select1st<std::pair<vtkm::cont::DeviceAdapterId const, (anonymous namespace
)::BufferState> >, std::less<vtkm::cont::DeviceAdapterId>, std::allocator<std::pair<vtkm::cont::DeviceAda
pterId const, (anonymous namespace)::BufferState> > >::_M_erase(std::_Rb_tree_node<std::pair<vtkm::cont::
DeviceAdapterId const, (anonymous namespace)::BufferState> >*) at Buffer.cxx:?

16: ./warpx() [0x120189d8]
    std::_Sp_counted_ptr<vtkm::cont::internal::Buffer::InternalsStruct*, (__gnu_cxx::_Lock_policy)2>::_M_
dispose() at ??:?

17: ./warpx() [0x12019294]
    vtkm::cont::internal::Buffer::~Buffer() at ??:?

18: ./warpx() [0x109ce36c]
    vtkm::rendering::raytracing::Ray<float>::~Ray() at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:98
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:108
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:136
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:206
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_vector.h:677
 (inlined by) ?? at /gpfs/alpine/csc340/world-shared/software/ascent/2021_09_01_gcc_9_3_0_warpx/summit/cuda/gnu/spack/opt/spack/linux-rhel8-power9le/gcc-9.3.0/vtk-m-1.6.0-zh4ljmhdg3xl7byw3uk6hui2urcvuncm/include/vtkm-1.6/vtkm/cont/ArrayHandle.h:376
 (inlined by) ?? at /gpfs/alpine/csc340/world-shared/software/ascent/2021_09_01_gcc_9_3_0_warpx/summit/cuda/gnu/spack/opt/spack/linux-rhel8-power9le/gcc-9.3.0/vtk-m-1.6.0-zh4ljmhdg3xl7byw3uk6hui2urcvuncm/include/vtkm-1.6/vtkm/rendering/raytracing/ChannelBuffer.h:45
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:98
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:108
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:136
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_construct.h:206
 (inlined by) ?? at /autofs/nccs-svm1_sw/summit/gcc/9.3.0-2/include/c++/9.3.0/bits/stl_vector.h:677
 (inlined by) vtkm::rendering::raytracing::Ray<float>::~Ray() at /gpfs/alpine/csc340/world-shared/software/ascent/2021_09_01_gcc_9_3_0_warpx/summit/cuda/gnu/spack/opt/spack/linux-rhel8-power9le/gcc-9.3.0/vtk-m-1.6.0-zh4ljmhdg3xl7byw3uk6hui2urcvuncm/include/vtkm-1.6/vtkm/rendering/raytracing/Ray.h:39

19: ./warpx() [0x10ff2a10]
    vtkm::rendering::MapperVolume::RenderCells(vtkm::cont::DynamicCellSetBase<vtkm::List<vtkm::cont::CellSetStructured<2>, vtkm::cont::CellSetStructured<3>, vtkm::cont::CellSetExplicit<vtkm::cont::StorageTagBasic, vtkm::cont::StorageTagBasic, vtkm::cont::StorageTagBasic>, vtkm::cont::CellSetSingleType<vtkm::cont::StorageTagBasic> > > const&, vtkm::cont::CoordinateSystem const&, vtkm::cont::Field const&, vtkm::cont::ColorTable const&, vtkm::rendering::Camera const&, vtkm::Range const&) at ??:?

20: ./warpx() [0x10a1aab0]
    vtkh::VolumeRenderer::RenderOneDomainPerRank() at ??:?

21: ./warpx() [0x10a1b2a4]
    vtkh::VolumeRenderer::Update() at ??:?

22: ./warpx() [0x10a12180]
    vtkh::Scene::Render() at ??:?

23: ./warpx() [0x1096c94c]
    ascent::runtime::filters::ExecScene::execute() at /tmp/cyrush/spack-stage/spack-stage-ascent-develop-we4tn3kj7xdbrwnsmaaf3kmncgbjufqe/spack-src/src/ascent/runtimes/flow_filters/ascent_runtime_rendering_filters.cpp:311
 (inlined by) ascent::runtime::filters::ExecScene::execute() at /tmp/cyrush/spack-stage/spack-stage-ascent-develop-we4tn3kj7xdbrwnsmaaf3kmncgbjufqe/spack-src/src/ascent/runtimes/flow_filters/ascent_runtime_rendering_filters.cpp:1572

24: ./warpx() [0x109b6b84]
    flow::Workspace::execute() at /tmp/cyrush/spack-stage/spack-stage-ascent-develop-we4tn3kj7xdbrwnsmaaf3kmncgbjufqe/spack-src/src/flow/flow_workspace.cpp:341

25: ./warpx() [0x10881ed8]
    ascent::AscentRuntime::Execute(conduit::Node const&) at /tmp/cyrush/spack-stage/spack-stage-ascent-develop-we4tn3kj7xdbrwnsmaaf3kmncgbjufqe/spack-src/src/ascent/runtimes/ascent_main_runtime.cpp:1659

26: ./warpx() [0x1086f578]
    ascent::Ascent::execute(conduit::Node const&) at /tmp/cyrush/spack-stage/spack-stage-ascent-develop-we4tn3kj7xdbrwnsmaaf3kmncgbjufqe/spack-src/src/ascent/ascent.cpp:448

27: ./warpx() [0x101b744c]
    FlushFormatAscent::WriteToFile(amrex::Vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >, amrex::Vector<amrex::MultiFab, std::allocator<amrex::MultiFab> > const&, amrex::Vector<amrex::Geometry, std::allocator<amrex::Geometry> >&, amrex::Vector<int, std::allocator<int> >, double, amrex::Vector<ParticleDiag, std::allocator<ParticleDiag> > const&, int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, bool, bool, bool, bool, bool, bool, bool, bool, bool, bool, bool, int, amrex::Geometry const&, bool) const at ??:?

28: ./warpx() [0x10101c50]
    FullDiagnostics::Flush(int) at ??:?
```

(google drive info to reproduce on slack)

**Pull request merged**